acs.service

```diff
@@ -2,7 +2,7 @@
 Description=Manages Clock Speed
 
 [Service]
-ExecStart=/home/cameron/.cargo/bin/acs run
+ExecStart=/home/[Your Directory Name Here]/.cargo/bin/acs run
```

Comment:
Thanks for the fix!
One thing, the service [setup script](https://github.com/JakeRoggenbuck/auto-clock-speed/blob/main/scripts/service_setup.sh) swaps "cameron" for "$USER", you could change this to "your-user-here" and then change the setup script from `sed -i "s/your-user-here/$USER/" $service` if you'd like

Reply:
We could also just do that later if you'd like

Reply:
O cool!
I'll do that in a bit

Reply:
Thanks!